src/sage/schemes/hyperelliptic_curves/hyperelliptic_g2.py

```diff
@@ -45,6 +45,15 @@ def jacobian(self):
             sage: f = x^5 - x^4 + 3
             sage: HyperellipticCurve(f).jacobian()
             Jacobian of Hyperelliptic Curve over Rational Field defined by y^2 = x^5 - x^4 + 3
+
+        TESTS:
+
+        Ensure that :issue:`37612` is fixed::
+
+            sage: R.<x> = QQ[]
+            sage: f = x^5 - x^4 + 3
+            sage: HyperellipticCurve(f).jacobian
+            <bound method HyperellipticCurve_g2.jacobian of Hyperelliptic Curve over Rational Field defined by y^2 = x^5 - x^4 + 3>
         """
```

Comment:
Instead you may test the type of the object returned by `HyperellipticCurve(f).jacobian()`. I think this is more to the point of the test.